src/test/java/pages/Homework22Page.java

```diff
@@ -0,0 +1,34 @@
+package pages;
+
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.By;
+
+public class Homework22Page extends BasePage
+{
+
+    public Homework22Page(WebDriver givenDriver)
+    {
+        super(givenDriver);
+    }
+    By doubleClickPlaylist = By.cssSelector(".playlist:nth-child(3)");
+    By changeNamePlaylist = By.cssSelector("[name='name']");
+    By actualDisplayedMessage = By.xpath("//div[@class=\"success show\"]");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```